service/systemd/service.go

```diff
@@ -230,8 +237,18 @@ func (s *Service) readConf() (common.Conf, error) {
 	var conf common.Conf
 
 	data, err := Cmdline{}.conf(s.Service.Name, s.DirName)
-	if err != nil {
+	errString := fmt.Sprintf("%v", err)
+	if err != nil && strings.Contains(errString, "No such file or directory") == false {
 		return conf, s.errorf(err, "failed to read conf from systemd")
+	} else if err != nil && strings.Contains(errString, "No such file or directory") == true {
```

Comment:
In this case you don't need the `strings.Contains(...)` - it's guaranteed to be true since we didn't go down the other path. If you want to keep it to be more explicit you can, but remove the `== true` at the end.